tests/unit/test_dynatrace.py

```diff
@@ -5,20 +5,66 @@
 from buildpack.telemetry import dynatrace
 from unittest import TestCase
 
+from buildpack.telemetry.dynatrace import _get_ingestion_url
+
 
 class TestDynatrace(TestCase):
     def test_is_enabled_false(self):
         # Should be False without necessary environment variables
-        self.assertFalse(dynatrace.is_enabled())
-
-    def test_get_ingestion_info(self):
-        # for get_ingestion_info(), also covers enabled case of is_enabled()
-        token = "DUMMY_TOKEN"
-        url = "DUMMY_URL"
-        expected_ingest_url = urljoin(url, dynatrace.INGEST_ENDPOINT)
-        env_vars = {"DT_PAAS_TOKEN": token, "DT_SAAS_URL": url}
-        with patch.dict(os.environ, env_vars):
-            actual_info = dynatrace.get_ingestion_info()
+        self.assertFalse(dynatrace.is_telegraf_enabled())
 
-        expected_info = (token, expected_ingest_url)
-        self.assertEqual(expected_info, actual_info)
+    def test_get_ingestion_url(self):
```

Comment:
Awesome! Nice to have these set of tests. 💯 

For the second URL, we could just refer to the constant in the file. So even if it is modified it still gets validated.

Reply:
Right, thats a good point.